src/esrs/state.rs

```diff
@@ -10,48 +11,49 @@ use crate::types::SequenceNumber;
 /// - a lock representing the atomicity of the access to the aggregate,
 /// - a state defined by the user of this library.
 pub struct AggregateState<S> {
-    pub(crate) id: Uuid,
-    pub(crate) sequence_number: SequenceNumber,
-    pub(crate) lock: Option<EventStoreLockGuard>,
-    pub(crate) inner: S,
-}
-
-impl<S: Clone> Clone for AggregateState<S> {
-    fn clone(&self) -> Self {
-        Self {
-            id: self.id,
-            sequence_number: self.sequence_number,
-            inner: self.inner.clone(),
-            lock: None,
-        }
-    }
+    id: Uuid,
+    sequence_number: SequenceNumber,
+    lock: Option<EventStoreLockGuard>,
+    inner: S,
```

Comment:
Fields are now private. All manipulation of this struct must happen through its public API.